### PR TITLE
cloud: ovirt: add snapshots module

### PR DESCRIPTION
**Pull request by machacekondra**
##### ISSUE TYPE

 - Feature Pull Request

##### COMPONENT NAME

ovirt_snapshots

##### ANSIBLE VERSION

```
2.3.0
```

##### SUMMARY

This PR adds one new module for managing oVirt snaphosts and one new module to retrive snapshots facts.

```

```

### CODE REVIEW

**Comment by machacekondra** [comment]
shipit

**Comment by machacekondra** [comment]
@ryansb

**Comment by mattclay** [comment]
CI failure due to missing docs fragment:

```
2017-02-01 19:27:21 ============================================================================
2017-02-01 19:27:21 lib/ansible/modules/cloud/ovirt/ovirt_snapshots_facts.py
2017-02-01 19:27:21 ============================================================================
2017-02-01 19:27:21 ERROR: DOCUMENTATION fragment missing: ovirt_snapshots
```

**Comment by machacekondra** [comment]
Thanks, fixed.

File: lib/ansible/modules/cloud/ovirt/ovirt_snapshots.py [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+       inconsistent."
+    - "Deleting a snapshot does not remove any information from the virtual
+       machine - it simply removes a return-point. However, restoring a virtual
+       machine from a snapshot deletes any content that was written to the
+       virtual machine after the time the snapshot was taken."
+extends_documentation_fragment: ovirt
+'''
+
+
+EXAMPLES = '''
```
Comment (ryansb) [inline lib/ansible/modules/cloud/ovirt/ovirt_snapshots.py:68]: In addition to examples, can you add docs on `RETURN` like what's in the `snapshot_facts` module below?

**Review by ryansb — CHANGES_REQUESTED** [review]

**Review by ryansb — APPROVED** [review]

**Comment by ryansb** [comment]
Looks good, just made some tiny changes to pass style checks. Once CI passes proper I can merge. 😄

**Pull request merged**